packages/core/src/admin-ui/components/__tests__/Pagination.test.tsx

```diff
@@ -1,122 +0,0 @@
-/**
- * @jest-environment jsdom
- */
-
-import React from 'react';
-import userEvent from '@testing-library/user-event';
-import { render, screen } from '@testing-library/react';
-import { Pagination } from '../Pagination';
-const useRouter = jest.spyOn(require('next/router'), 'useRouter');
-describe('<Pagination/>', () => {
-  it('should render a nav element with role navigation', () => {
-    useRouter.mockImplementation(() => ({
```

Comment:
I think that the cost of fixing these tests (now and over time) would be more costly than the benefit of the tests so I just deleted them